addons/web/static/src/js/widgets/colorpicker.js

```diff
@@ -61,11 +66,19 @@ var Colorpicker = Dialog.extend({
         this.$colorpickerPointer = this.$('.o_picker_pointer');
         this.$colorSlider = this.$('.o_color_slider');
         this.$colorSliderPointer = this.$('.o_slider_pointer');
+        this.$colorOpacity = this.$('.o_opacity_slider');
+        this.$colorOpacityPointer = this.$('.o_opacity_pointer');
+
+        // Add transparent image on color preview
+        this.$('.o_color_preview').parent().css('background-image', "url('/web_editor/static/src/img/transparent.png'");
 
         var defaultColor = this.options.defaultColor || '#FF0000';
-        var rgb = defaultColor.match(/rgb\((\d{1,3}), ?(\d{1,3}), ?(\d{1,3})\)/);
-        if (rgb) {
-            this._updateRgb(parseInt(rgb[1]), parseInt(rgb[2]), parseInt(rgb[3]));
+        var rgba = defaultColor.match(/^rgba?\((\d+),\s*(\d+),\s*(\d+)(?:,\s*(\d+(?:\.\d+)?))?\)$/);
+        if (rgba) {
+            if (isNaN(rgba[4])) {
```

Comment:
I don't think it will ever be `NaN` here, probably `undefined` or `null`

Reply:
It is indeed possible to be NaN : https://drive.google.com/open?id=1KJSOfHnMPmruOHl7YLkFb46g29jPoNwx. Is it a problem if it stays ?

Reply:
It appears as NaN there because you parse the value as a float later... but it is not NaN at that moment.